common/app/model/content.scala

```diff
@@ -66,7 +67,8 @@ final case class Content(
   showByline: Boolean,
   hasStoryPackage: Boolean,
   rawOpenGraphImage: String,
-  showFooterContainers: Boolean = false
+  showFooterContainers: Boolean = false,
+  blocks: Option[Blocks] = None
```

Comment:
@johnduffell Do you have any insight into `blocks` from the work you did on liveblogs ? Does this seem a logical use to you ?